src/actions/payments.js

```diff
@@ -75,8 +75,7 @@ class ActionsPayments {
           pay_req: paymentRequest,
         })
         .then(response => {
-          resolve(response.num_satoshis);
-          // resolve(response);
+          resolve(response);
```

Comment:
What's the reasoning behind changing the return value?

Reply:
To be able to access both `num_satoshis` and `description` from the view

Reply:
Ok. We should probably stay consistent with the return values in other actions and map to camel case like in `action/channels.js` In that case.